sotodlib/io/imprinter.py

```diff
@@ -1222,20 +1222,37 @@ def update_bookdb_from_g3tsmurf(
                                 np.min([o.stop for o in obs_list]) - 
                                 np.max([o.start for o in obs_list])
                             )
-
+                            
                             if overlap_time.total_seconds() < 0:
                                 continue
                             if overlap_time.total_seconds() < min_overlap:
                                 continue
-                            # add all of the possible overlaps
-                            output.append(
-                                ObsSet(
-                                    obs_list,
-                                    mode="obs",
-                                    slots=self.tubes[tube]["slots"],
-                                    tel_tube=tube,
+
+                            if np.all([o.timing for o in obs_list]):
```

Comment:
Shouldn't one split the obs_list into two lists?:
- the ones with timing can be bound, _together_, into a single book
- the ones without must be individually bound

Someday we might have one slot being weird or misconfigured; need not throw out everything in that case.

Reply:
makes sense. From a configuration standpoint I think it's more of an all or nothing thing. But easy enough to fix.